### PR TITLE
added test for sort_numbers in a new repo (didn't find an existing on…

### PR DESCRIPTION
**Pull request by AGuyNextDoor**
…e..)

### CODE REVIEW

**Comment by AGuyNextDoor** [comment]
https://github.com/fewlinesco/sparta_courses/pull/325

File: 06_react_redux/02_theory/02_sort_numbers/__tests__/sortNumbers.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+})
+
+it("the first printed list is in ascending order", () => {
+  const listMock = shallow(<App/>);
+  const stateList = listMock.state("listItem");
+
+  let beforeClick = [];
+  stateList.forEach(element => {
+    beforeClick.push(element.props.children)
+  })
```
Comment (Fenntasy) [inline 06_react_redux/02_theory/02_sort_numbers/__tests__/sortNumbers.test.js]: you could do a `map` here:

```javascript
const beforeClick = stateList.map(element => element.props.children);
```

File: 06_react_redux/02_theory/sort_numbers/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sort_numbers",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "enzyme-adapter-react-16": "^1.7.1",
```
Comment (Fenntasy) [inline 06_react_redux/02_theory/sort_numbers/package.json]: forgot `enzyme` there

Comment (AGuyNextDoor) [reply 06_react_redux/02_theory/sort_numbers/package.json]: done !

**Review by Fenntasy — APPROVED** [review]

**Pull request merged**